docs/proposals/20210310-opt-in-autoscaling-from-zero.md

```diff
@@ -204,8 +245,16 @@ status:
     memory: 500mb
     cpu: "1"
     nvidia.com/gpu: "1"
+  nodeInfo:
+    architecture: arm64
+    operatingSystem: linux
 ```
 
+The information stored in the `status.nodeInfo` field is rendered as labels for the node object that is created within
```

Comment:
q: why the autoscaler should render those info as a label on nodes, when this proposal is about autoscale from 0 (so for when there no nodes).

Reply:
the node I meant is the simulated one that the autoscaler creates in its logic to execute the scheduler simulator. What we expect in this case is that the pod triggering the autoscaler has a node selector or node affinity about the `kubernetes.io/arch` label and we have to provide the same label on the (simulated) node side to let the scheduler be aware that adding that node would satisfy the scheduling predicates.

Reply:
I think this is resolved via https://github.com/kubernetes-sigs/cluster-api/pull/11962#discussion_r2073325232